authorization/src/main/kotlin/com/ritense/authorization/AuthorizationRequest.kt

```diff
@@ -19,4 +19,5 @@ package com.ritense.authorization
 interface AuthorizationRequest<T> {
     val resourceType: Class<T>
     val action: Action<T>
+    val user: String?
```

Comment:
I couldn't find any constructor usage where this `user` property is used. Why is it there?

Reply:
Its only used [here](https://github.com/valtimo-platform/valtimo-backend-libraries/pull/780/files#diff-e820a081bd1a17e1df5c297fc16e03563c11d8c99e42e62406e43be5f1f43e0aR164).

Its to check this scenario:
When the method `.assign(assignee)` is called, the `assignee` must have the `ASSIGNABLE` permission.

Reply:
This has been refactored into a specific AuthorizationRequest type when requesting permissions for another user. See DelegateUserEntityAuthorizationRequest